data/index.json

```diff
@@ -1,7 +1,7 @@
 {
   "chapter": "Los Angeles",
   "state": "California",
-  "eventAddress": "TBA",
+  "eventAddress": "2150 Park Place, Suite 100, El Segundo, CA 90245",
   "eventDate": "November 4th, 2017",
   "eventDetails": "<h2 class=\"u-color--light\">Before the Event</h2><p class=\"u-color--light\">Before the event on Saturday, install Node.js and a workshop that you'd like to work on. If you are just starting out, \"learnyounode\" is highly recommended. Doing this before the event will help you get going quickly.</p><h2 class=\"u-color--light\">Parking and Event time</h2><p class=\"u-color--light\">TBA</p><h2 class=\"u-color--light\">Check-in and Food / Drink</h2><p class=\"u-color--light\">Please find a name tag and write your name. Food and drink should be available so please help yourself before the event begins.</p><h2 class=\"u-color--light\">Tables and Wifi</h2><p class=\"u-color--light\">Please find a table and chair inside the main training room. Connect yourself to the wireless network and download any necessary programs and workshopper to get started (Node.js and Learnyounode if it's your first time with Node.js). The network name will displayed on the video screens. Mentors will be available to assist if you haven't done this step prior to arriving at the event.</p><h2 class=\"u-color--light\">What to Expect</h2><p class=\"u-color--light\">Actual time to get to work on a workshopper will be about 3 to 3 1/2 hours. You are free to work on any workshopper you like and mentors will be available if you get stuck or need any assistance. Everyone is here to learn so there are no egos and no questions that are too silly to ask. Come join and have fun while learning!</p>",
```

Comment:
there is a paragraph to add to this line, it is a total pain in the ass.

you can look at the diff of what I removed for guidance, https://github.com/nodeschool/los-angeles/commit/17d4e2f82d97c44d51ff28ad284db84259b8e462

Reply:
Yeah I went based off the diff when I did this and somehow totally missed that lol. Nice catch.

Reply:
yeah, normally we don't have to change anything but the date so it is really simple.

this time is kinda weird